image/templates/helm/stackrox-secured-cluster/templates/service-ca.yaml

```diff
@@ -13,4 +13,7 @@ metadata:
 type: Opaque
 stringData:
   ca.pem: |
+    {{- if hasKey (.Values.metadata).annotations "init-bundle.stackrox.io/name" }}
+    {{- fail "It looks like you're using the incorrect init-bundle format. Make sure that you're using an init-bundle that was formatted for the Helm installation method and not Operator installation method." | indent 4 }}
```

Comment:
We have a macro named `srox.fail`, which we are using in these kind of situations. How about using that as well, for consistency reasons?

Reply:
Thinking about this a little bit more, I am wondering if we should really fail. We could also emit a warning using `srox.warn`.

Reasoning: Strictly speaking failing in this situation would be a breaking change. There *could* be users who, let's say, inject *both* bundles into the Helm chart rendering. Of course, that would be somewhat weird, because only one of them would be effective. But, after this change, such a process would suddenly fail.

What do you think about warning instead and rephrasing the message a little bit. Something along the lines

> It seems an init bundle in the operator format has been provided. Note that this bundle format is ignored by the Helm chart. Make sure to provide an init bundle for Helm chart installation.